### PR TITLE
Exporters expect view tags to be sorted

### PR DESCRIPTION
**Pull request by kyledj**
I noticed tags coming through incorrectly from the Prometheus exporter. That exporter (and possibly others) [expect view tags to be sorted](https://github.com/census-instrumentation/opencensus-go/blob/7a9460ceb1a16caeaedb3d40fcc77176ea9713b2/exporter/stats/prometheus/prometheus.go#L249) since the [collector seems to explicitly sort them in rows](https://github.com/census-instrumentation/opencensus-go/blob/7635ef593d5ab616c7708ba9fe28bb4465d2878e/stats/collector.go#L86)

I could do this from the exporter itself, but I don't think there is any downside to doing it here, keeping the exporter assumption intact.

### CODE REVIEW

File: stats/view_test.go [hunk 1]
```diff
@@ -815,6 +815,31 @@ func Test_View_MeasureFloat64_AggregationMean_WindowCumulative(t *testing.T) {
...
 
+func TestViewSortedKeys(t *testing.T) {
+	k1, _ := tag.NewKey("a")
+	k2, _ := tag.NewKey("b")
+	k3, _ := tag.NewKey("c")
+	ks := []tag.Key{k1, k3, k2}
+
+	v, err := NewView("sort_keys", "desc sort_keys", ks, nil, MeanAggregation{}, Cumulative{})
+	if err != nil {
+		t.Fatalf("sort_keys failed creating view: %v", err)
```
Comment (rakyll) [inline stats/view_test.go]: "NewView() = %v", err

File: stats/view_test.go [hunk 2]
```diff
@@ -815,6 +815,31 @@ func Test_View_MeasureFloat64_AggregationMean_WindowCumulative(t *testing.T) {
...
+	}
+
+	expected := []string{"a", "b", "c"}
```
Comment (rakyll) [inline stats/view_test.go]: want := []string{...}

File: stats/view_test.go [hunk 3]
```diff
@@ -815,6 +815,31 @@ func Test_View_MeasureFloat64_AggregationMean_WindowCumulative(t *testing.T) {
...
+	vks := v.TagKeys()
+	if len(vks) != len(expected) {
+		t.Fatalf("unexpected keys: %+v, expected %+v", vks, expected)
```
Comment (rakyll) [inline stats/view_test.go]: t.Errorf("Keys = %+v; want %+v", vks, want)

File: stats/view_test.go [hunk 4]
```diff
@@ -815,6 +815,31 @@ func Test_View_MeasureFloat64_AggregationMean_WindowCumulative(t *testing.T) {
...
+	}
+
+	for i, v := range expected {
+		if v != vks[i].Name() {
+			t.Fatalf("expected %s, got %s", v, vks[i].Name())
+		}
+	}
+
```
Comment (rakyll) [inline stats/view_test.go]: Remove line.

File: stats/view_test.go [hunk 5]
```diff
@@ -815,6 +815,31 @@ func Test_View_MeasureFloat64_AggregationMean_WindowCumulative(t *testing.T) {
...
+	expected := []string{"a", "b", "c"}
+	vks := v.TagKeys()
+	if len(vks) != len(expected) {
+		t.Fatalf("unexpected keys: %+v, expected %+v", vks, expected)
+	}
+
+	for i, v := range expected {
+		if v != vks[i].Name() {
+			t.Fatalf("expected %s, got %s", v, vks[i].Name())
```
Comment (rakyll) [inline stats/view_test.go]: if got, want := v, vks[i].Name(); got != want {
t.Errorf("View name = %q; want %q", got, want)
}

**Comment by kyledj** [comment]
Updated

**Pull request merged**